build/Targets/Imports.targets

```diff
@@ -470,4 +471,23 @@
 
   <!-- Returns the current build version. Used in .vsixmanifests to substitute our build version into them -->
   <Target Name="GetBuildVersion" Outputs="$(VsixVersion)" />
+
+  <!-- 
+    In order to leverage LUT testing we need to have both of the follownig packages in 
+    our build.  
+
+        - xunit.runner.desktop
+        - xunit
+
+    Both of these include xunit.abstractions.dll in different forms: content and reference
+    respectively.  This creates a double write violation in our build.  The content item
+    is unnecessary for us since we reference the xunit package hence remove it here. 
+
+    https://github.com/dotnet/roslyn/issues/18753
+  -->
+  <Target Name="RemoveDuplicateXUnitContent">
+    <ItemGroup>
+      <Content Remove="@(Content)" Condition="'%(Filename)%(Extension)' == 'xunit.abstractions.dll'" />
```

Comment:
Adding Inputs/Outputs would break the correctness of this target. Imagine you do a build, everything completes. Then a build the second time and now your target *won't* run, which means you'll end up being in the bad state. You might end up in some fun situation where then every other build would be "right" and then "wrong".

Another way to think about it: Inputs/Outputs are a statement that "this target consumes the _contents_ of those files"...this target doesn't.